users/manna-harbour_miryoku/manna-harbour_miryoku.c

```diff
@@ -88,4 +88,15 @@ combo_t key_combos[COMBO_COUNT] = {
   #endif
   COMBO(thumbcombos_fun, KC_APP)
 };
+
+layer_state_t default_layer_state_set_user(layer_state_t state) {
+  if (get_highest_layer(state) == U_TAP) {
+    combo_disable();
+  } else {
+    combo_enable();
+  }
+
+  return state;
+}
```

Comment:
`U_TAP` may not be defined if https://github.com/manna-harbour/miryoku/discussions/85 is used to remove or rename the layer.

Wrapping with `#if defined (U_TAP)` / `#endif` should deal with that.